static/js/tippyjs.js

```diff
@@ -250,7 +250,7 @@ export function initialize() {
     });
 
     delegate("body", {
-        target: ".narrow_to_compose_recipients",
+        target: [".narrow_to_compose_recipients"],
```

Comment:
Is this change necessary?

Reply:
It is not, this was a left over from a previous approach. This has been removed in the latest commit